src/buyingcatalogue/Chart.yaml

```diff
@@ -69,7 +69,7 @@ dependencies:
   repository: "@gpitfuturesdevacr"
 - name: mp
   condition: mp.enabled
-  version: 1.26.3
+  version: 1.26.5
```

Comment:
We wouldn't usually update the core component versions outside of a release

Reply:
I'll revert this change